.gitignore

```diff
@@ -18,3 +18,5 @@ profile
 DerivedData
 
 local.properties
+
+.idea/*
```

Comment:
Sorry, I'm new to project contributions, do I have to make a new pull request to edit it or is there a way to edit it here?